### PR TITLE
[HttpFoundation] document `Request::getPayload()`

### LINKED ISSUE #18234: [HttpFoundation] add `Request::getPayload()`
| Q            | A
| ------------ | ---
| Feature PR   | symfony/symfony#49614
| PR author(s) | @kbond
| Merged in    | 6.3

We created this issue to not forget to document this new feature. We would really appreciate if you can help us with this task. If you are not sure how to do it, please ask us and we will help you.

To fix this issue, please create a PR against the 6.3 branch in the [symfony-docs repository](https://github.com/symfony/symfony-docs).

Thank you! :smiley:

### PR DESCRIPTION
**Pull request by kbond**
Closes #18234.

### CODE REVIEW

File: components/http_foundation.rst [hunk 1]
```diff
@@ -216,6 +216,17 @@ If the request body is a JSON string, it can be accessed using
 
     $data = $request->toArray();
 
+If the request data could be $_POST data *or* a JSON string, you can use
+:method:`Symfony\\Component\\HttpFoundation\\Request::getPayload`
```
Comment (OskarStark) [inline components/http_foundation.rst]:
```suggestion
the :method:`Symfony\\Component\\HttpFoundation\\Request::getPayload` method
```

File: components/http_foundation.rst [hunk 2]
```diff
@@ -216,6 +216,17 @@ If the request body is a JSON string, it can be accessed using
...
+which returns an instance of :class:`Symfony\\Component\\HttpFoundation\\InputBag`
+wrapping this data::
+
+    $data = $request->getPayload();
+
+.. versionadded:: 6.3
+
+    The ``Request::getPayload()`` method was introduced in Symfony 6.3.
```
Comment (OskarStark) [inline components/http_foundation.rst]:
```suggestion
    The :method:`Symfony\\Component\\HttpFoundation\\Request::getPayload`
    method was introduced in Symfony 6.3.
```

File: components/http_foundation.rst [hunk 3]
```diff
@@ -216,6 +216,17 @@ If the request body is a JSON string, it can be accessed using
 
     $data = $request->toArray();
 
+If the request data could be $_POST data *or* a JSON string, you can use
```
Comment (OskarStark) [inline components/http_foundation.rst]:
```suggestion
If the request data could be ``$_POST`` data *or* a JSON string, you can use
```

**Comment by javiereguiluz** [comment]
Thanks Kevin.

**Pull request merged**